awx/ui/client/src/templates/job_templates/job-template.form.js

```diff
@@ -257,6 +257,19 @@ function(NotificationsList, i18n) {
                     dataPlacement: 'right',
                     control: '<instance-groups-multiselect instance-groups="instance_groups" field-is-disabled="!(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)"></instance-groups-multiselect>',
                 },
+                job_slice_count: {
+                    label: i18n._('Job Slicing'),
+                    type: 'number',
+                    integer: true,
+                    min: 1,
+                    default: 1,
+                    spinner: true,
+                    dataTitle: i18n._('Slice Job Count'),
+                    dataPlacement: 'right',
+                    dataContainer: 'body',
+                    awPopOver: "<p>" + i18n._("The number of jobs to slice into at runtime. Will cause the Job Template to launch a workflow if value is greater than 1.") + "</p>",
```

Comment:
@one-t @wenottingham As a documentation matter, we were discussing that this text might need to say something about the intended use case. The intended use case is that the number of slices is almost equal to, or exactly equal to, the number of instances available to run the job template. Logic is then that the job distributes throughout the cluster, or part of the cluster.

We don't want to say all of that, but some mention of number of available instances would seem to address some of QEs concerns about the fact that it is up to the user, themselves, to size this correctly.

Reply:
> For best behavior, this number should be high enough that each slice is smaller than an individual cluster node capacity.

For best behavior, this number should correspond to the number of execution instances available to this template.

Reply:
maybe "execution instances" -> "execution nodes"... iffy about that

Reply:
Surely having more, smaller jobs (50 slices, even if you have only 5 nodes) should be fine. @matburt ?

Reply:
Why are we even talking about slices relative to number of execution nodes? It's entirely possible that you'd end up with multiple jobs on execution nodes. This isn't the kind of thing we need to get into because it'll end up being different for any individual customer's use case. Bill's statement in the 2nd comment is fine.

Reply:
> The number of job slices to create for this job. This should not be set for jobs where cross-host orchestration is done. For best behavior, this number should be high enough that each slice is smaller than an individual cluster node capacity.

Okay, so this is what we want to go with.

Problem, this kind of drops the context of:

> Will cause the Job Template to launch a workflow if value is greater than 1.

So maybe

> The number of job slices to create for this job. Will cause the Job Template to launch a workflow if value is greater than 1. This should not be set for jobs where cross-host orchestration is done. For best behavior, this number should be high enough that each slice is smaller than an individual cluster node capacity.

This seems reasonable, when compared to other tooltips

![screen shot 2018-10-30 at 9 03 04 am](https://user-images.githubusercontent.com/1385596/47719816-ad57a080-dc22-11e8-95c8-6344ee428ca0.png)

Reply:
I'd drop the bit about a workflow... that's (essentially) an implementation detail that we are surfacing, and we could decide not to in the future.